src/NuGet.Core/NuGet.Packaging/Signing/SignatureException.cs

```diff
@@ -10,6 +10,7 @@
 
 namespace NuGet.Packaging.Signing
 {
+    [SerializableAttribute]
     public class SignatureException : PackagingException
```

Comment:
This class needs to implement `ISerializable` properly for all the properties to round trip.